### PR TITLE
Some corrections

### PR DESCRIPTION
**Pull request by iddm**

### CODE REVIEW

**Comment by vityafx** [comment]
```
-As a proxy object, stream should never outlive its source. 
+As a proxy object, stream should never outlive it's source. 
```

Should be changed back to `its`.

File: Readme.md [hunk 1]
```diff
@@ -52,16 +52,16 @@ If you are curious the result is:
 
 ## Under the hood ##
 Streams are designed to be fast and lightweight proxy objects. Streams:
```
Comment (randrewy) [inline Readme.md]: Note, I mean 'Streams library' here, never thought it can be not so clear.

**Comment by randrewy** [comment]
Thanks for pointing that out. Please take a look at my comments, and edit your request.
Waiting for another :gb: pull

File: Readme.md [hunk 2]
```diff
@@ -52,16 +52,16 @@ If you are curious the result is:
...
-- doesn't own the underlying collection; 
-- doesn't modify the underlying collection; 
-- doesn't allocate memory on the heap;
-- never throws exceptions unless it's thrown from inside user code;
+- don't own the underlying collection;
```
Comment (randrewy) [inline Readme.md]: Note, I mean 'Streams library:' here, never thought it can be not so clear.

**Pull request closed without merging**